client/src/components/Panels/SidePanel.vue

```diff
@@ -26,8 +26,11 @@ export default {
     props: ["currentPanel", "currentPanelProperties", "side"],
     data() {
         return {
-            show: true,
+            show: localStorage.getItem(`${this.side}_panel_show`)
+                ? localStorage.getItem(`${this.side}_panel_show`) === "true"
```

Comment:
As mentioned I suggest to just use this term here. Maybe prefix with `galaxy_` if we go with directly storing in the local storage.